packages/amplication-client/src/Workspaces/WorkspaceHeader/GitHubBanner.tsx

```diff
@@ -39,9 +38,9 @@ export default function GitHubBanner() {
         target="_blank"
         rel="noreferrer"
         className={`${CLASS_NAME}__banner__cta`}
-        onClick={() => {
-          handleBannerClick(AnalyticsEventNames.StarUsBannerCTAClick);
-        }}
+        // onClick={() => {
+        //   handleBannerClick(AnalyticsEventNames.StarUsBannerCTAClick);
+        // }}
```

Comment:
The callback should still be executed.
do not use arrow function - just use the name of the callback

onClick={handleStarUsClick}

Reply:
also, please fix the other button onClick event to use the callback name instead of an arrow function

onClick={handleCloseStarUsClick}